internal/core/algorithm/ngt/option_test.go

```diff
@@ -348,9 +350,9 @@ func TestWithDimension(t *testing.T) {
 			},
 		},
 		{
-			name: "set success when the size is 65536",
+			name: "set success when the size is maxDim",
 			args: args{
-				size: 65536,
+				size: 1<<32 - 1,
```

Comment:
```suggestion
				size: maxDim,
```
Why don't you use `maxDim`?